frontend/src/components/Teams/MyTeams/partials/CardTeam/BoardsInfo.tsx

```diff
@@ -1,25 +1,20 @@
-import { useQuery } from 'react-query';
 import Link from 'next/link';
 
-import { getDashboardHeaderInfo } from 'api/authService';
 import Icon from 'components/icons/Icon';
 import Flex from 'components/Primitives/Flex';
 import Text from 'components/Primitives/Text';
+import { Team } from '../../../../../types/team/team';
 
 type BoardsInfoProps = {
 	userSAdmin: boolean | undefined;
 	teamAdmin: boolean;
+	team: Team;
 };
 
-const BoardsInfo = ({ userSAdmin, teamAdmin }: BoardsInfoProps) => {
-	const { data: boardInfo } = useQuery('dashboardInfo', () => getDashboardHeaderInfo(), {
-		enabled: true,
-		refetchOnWindowFocus: false
-	});
-
-	if (!boardInfo?.boardsCount) {
+const BoardsInfo = ({ userSAdmin, teamAdmin, team }: BoardsInfoProps) => {
+	if (team.boardsCount === 0) {
 		return (
-			<Flex css={{ ml: '$40', display: 'flex', alignItems: 'center' }}>
+			<Flex css={{ ml: '$20', display: 'flex', alignItems: 'center' }}>
```

Comment:
```suggestion
			<Flex css={{ ml: '$20', alignItems: 'center' }}>
```
The styled `flex` already has the `display: 'flex'`